Dockerfile

```diff
@@ -13,20 +11,20 @@ ENV     RUSTFLAGS="-C target-feature=-crt-static"
 
 COPY    . .
 RUN     set -eux; \
-        apkArch="$(apk --print-arch)"; \
-        if [ "$apkArch" = "aarch64" ]; then \
+        arch="$(dpkg --print-architecture)"; \
+        if [ "$arch" = "aarch64" ]; then \
             export JEMALLOC_SYS_WITH_LG_PAGE=16; \
         fi && \
         cargo build --release
 
 # Run
-FROM    alpine:3.16
+FROM    debian:11.6
 
 ENV     MEILI_HTTP_ADDR 0.0.0.0:7700
 ENV     MEILI_SERVER_PROVIDER docker
 
-RUN     apk update --quiet \
-        && apk add -q --no-cache libgcc tini curl
+RUN     apt update -q \
+        && apt install -q -y tini
```

Comment:
```suggestion
RUN     set -ex; \
        apt-get update -q; \
        apt-get install -q -y --no-install-recommends tini; \
        rm -rf /var/lib/apt/lists/*
```

Reply:
the `&&` vs `;` is actually preference but I thought why not also use the `;` syntax here as its used in the build step as well

Reply:
Thanks @mmachatschek 

What is the purpose of the last line? `rm -rf /var/lib/apt/lists/*`

Reply:
The directory is filled with information about packages and versions. In docker builds this is removed to reduce the size of the final image.

(This can be looked up in other docker files that build Debian/Ubuntu images.)